lisa/platform_.py

```diff
@@ -22,6 +25,23 @@
 
 _get_init_logger = partial(get_logger, "init", "platform")
 
+PlatformStatus = Enum(
+    "TestRunStatus",
+    [
+        "INITIALIZED",
+        "PREPARED",
```

Comment:
"ENVIRONMENT_PREPARED", "ENVIRONMENT_DEPLOYED", "ENVIRONMENT_DELETED". And in the future, it will bring the environment name or more information.

Reply:
Makes sense! I will update this.